### DISCUSSION TITLE
您好，为何 ant-design 组件库官网没有一个返回到顶部的功能？

### DISCUSSION THREAD

**Discussion by xsqy** [ant-design/ant-design#33305] [Q&A] [answered]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
在查看表单组件或表格组件的API部分后，想使用顶部搜索框的话需要手动滑动滚轮至页面顶部而且页面内容较多,滑动时间较长。使用起来不太方便。

### What does the proposed API look like?
无

**Comment by afc163** [comment, marked as answer]
<kbd>cmd</kbd> + <kbd>K</kbd> 或者 <kbd>/</kbd> 都可以直接进行搜索。

**Reply by xsqy** [reply]
好的

**Comment by 2048920439** [comment]
emmmm,都2023了,我也在找除了直接操作DOM还有什么办法能主动回到顶部

**Comment by MadCcc** [comment]
Header 已经 sticky 了，还有需要回到顶部的理由么

**Reply by 2048920439** [reply]
更新整个table数据之后,最新数据一般都会出现在table的最前面,所有回到顶部用户体验会更好
